rogue-thi-app/pages/food.js

```diff
@@ -218,10 +237,29 @@ export default function Mensa () {
       </AppNavbar>
 
       <AppBody>
-        <ReactPlaceholder type="text" rows={20} ready={foodDays}>
-          <SwipeableTabs className={styles.tab}>
-            {foodDays && foodDays.map((day, idx) => renderMealDay(day, idx))}
-          </SwipeableTabs>
+        <div className={styles.weekSelector}>
+          <Button className={styles.prevWeek} variant="link" onClick={() => setPage(0)} disabled={page === 0}>
+            <FontAwesomeIcon title="Woche zurück" icon={faChevronLeft} />
+          </Button>
+          <div className={styles.currentWeek}>
+            {page === 0 && WORD_THIS_WEEK}
+            {page === 1 && WORD_NEXT_WEEK}
+          </div>
+          <Button className={styles.nextWeek} variant="link" onClick={() => setPage(1)} disabled={page === 1}>
+            <FontAwesomeIcon title="Woche vor" icon={faChevronRight} />
+          </Button>
+        </div>
+
+        <ReactPlaceholder type="text" rows={20} ready={currentFoodDays && futureFoodDays}>
+          <SwipeableViews index={page} onChangeIndex={idx => setPage(idx)}>
+            <SwipeableTabs className={styles.tab} fillers={dayFiller}>
+              {currentFoodDays && currentFoodDays.map((day, idx) => renderMealDay(day, idx))}
+            </SwipeableTabs>
+
+            <SwipeableTabs className={styles.tab}>
```

Comment:
If the entire next week is not available, such as with holidays, I would add the fillerdays here as well for design consistency.

Reply:
Due to the `daysTillFriday` variable being set to zero if no data is available, the fillers will be added.